practices/semver.md

```diff
@@ -31,13 +31,6 @@ When a piece of software under heavy development many major/minor releases may h
 What _might_ be a worry is if your version looked like 2.1.45 - did you really release 45 patches/bug fixes in a row without adding any new features?
 
 ## Find out more:
-Canonical source of info: http://semver.org/
-How we version and release our open-source software: https://github.com/springernature/shunter/blob/master/docs/developer-guide.md#versioning
-More reading: https://abdulapopoola.com/2015/10/26/what-is-semver/
-
-
-
-
-
-
-
+- Canonical source of info: http://semver.org/
+- [How we version and release our open-source software](https://github.com/springernature/shunter/blob/master/docs/developer-guide.md#versioning)
+- More reading: https://abdulapopoola.com/2015/10/26/what-is-semver/
```

Comment:
rather than having some text, with a url following, can we instead have sensible linked text? Is more accessible.